bin/compile

```diff
@@ -7,24 +7,18 @@ BUILD_DIR=$1
 CACHE_DIR=$2
 BIN_DIR=$(cd $(dirname $0); pwd) # absolute path
 
-GETTEXT_TAR="http://ftp.gnu.org/pub/gnu/gettext/gettext-0.18.3.1.tar.gz"
-GETTEXT_HOME=$BUILD_DIR/.gettext
+GETTEXT_TAR="http://ftp.gnu.org/pub/gnu/gettext/gettext-0.19.8.1.tar.gz"
 GETTEXT_CACHE=$CACHE_DIR/.gettext
 
-PATH=$GETTEXT_HOME/bin:$PATH
-LD_LIBRARY_PATH=$GETTEXT_HOME/lib:$LD_LIBRARY_PATH
+PATH=$GETTEXT_CACHE/bin:$PATH
```

Comment:
perhaps this should be a env/setting somewhere? 

I can imagine people using this buildpack and wanting to have the binaries